src/tree_sitter/parser.h

```diff
@@ -13,8 +13,9 @@ extern "C" {
 #define ts_builtin_sym_end 0
 #define TREE_SITTER_SERIALIZATION_BUFFER_SIZE 1024
 
-#ifndef TREE_SITTER_API_H_
```

Comment:
Can you check out tree-sitter from master or 0.20.9 and install the cli locally, or run `cargo install --git https://github.com/tree-sitter/tree-sitter`? We will have 0.20.9 on npm/crates early next week, but for now I'd like to keep generated output as it is on 0.20.9, sorry for the trouble